packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -166,6 +166,68 @@ export class RestAPIClient {
 		return response.json();
 	}
 
+	/**
+	 * Gets the latest of post from the blogs a user follows.
+	 *
+	 *
+	 * @returns {Promise<ReaderParams}.
```

Comment:
**nit**:the type declaration is missing the closing angular bracket.

Reply:
Good catch! Just fixed it.